pycroft/helpers/user.py

```diff
@@ -12,8 +12,9 @@
 
 
 def generate_password(length):
-    charset = "abcdefghijklmnopqrstuvwxyz!$%&()=.," \
-              ":;-_#+1234567890ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    #without hard to distinguish characters l/1 0/O
```

Comment:
Thanks! I will replace this comment by a docstring in a few minutes though.
By the way: [PEP8 requires](https://www.python.org/dev/peps/pep-0008/#inline-comments) exactly one space after the `#` (and at least two spaces before it if you append the comment to a line.)